os-ballerina/build.gradle

```diff
@@ -255,7 +250,7 @@ publishing {
     repositories {
         maven {
             name = "GitHubPackages"
-            url = uri("https://maven.pkg.github.com/ballerina-platform/module-ballerina-os")
+            url = uri("https://maven.pkg.github.com/ballerina-platform/module-${packageOrg}-${packageName}")
```

Comment:
I don't see any advantage of parameterizing this GitHub URL. Even for some modules, the URL and the `packageOrg` are different. Therefore, we have to be careful when we change them like this.

Reply:
How can the URL and the packageOrg is different? That's weird. The reason to add this here is to remove the hard-coded strings as much as possible.